src/components/Business/Business.js

```diff
@@ -3,16 +3,19 @@ import "./Business.css";
 
 class Business extends React.Component {
   render() {
-    const urlstring = `https://www.google.com/maps/search/?api=1&query=${this.props.business.name} ${this.props.business.address}`.replace(/ /g, "+");
+    const addressurl = `https://www.google.com/maps/search/?api=1&query=${this.props.business.name} ${this.props.business.address}`.replace(/ /g, "+");
+    const websiteUrl = this.props.business.url;
     return (
       <div className="Business">
         <div className="image-container">
-          <img src={this.props.business.imageSrc} alt="" />
+          <a href={websiteUrl} target="_blank" rel="noopener noreferrer">
```

Comment:
Great, use of rel=`noopener` and `noreferrer` here. I appreciate it @adgai19. Thanks for keeping the security in mind.